de.blobbyvolley.BlobbyVolley2.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>de.blobbyvolley.BlobbyVolley2.desktop</id>
```

Comment:
```suggestion
  <id>de.blobbyvolley.BlobbyVolley2</id>
```
Avoid adding unnecessary .desktop suffix to the app-id. Instead add a launchable tag `<launchable type="desktop-id">de.blobbyvolley.BlobbyVolley2.desktop</launchable>

Reply:
I hope I fixed this. :)